AmahiAnywhere/AmahiAnywhere/SettingTableViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  SettingTableViewController.swift
+//  AmahiAnywhere
+//
+//  Created by Syed on 21/02/18.
+//  Copyright © 2018 Amahi. All rights reserved.
+//
+
+import UIKit
+import MessageUI
+
+class SettingTableViewController: UITableViewController ,MFMailComposeViewControllerDelegate {
+    
+    
+    //MARK:- Mail Configuration
+    
+    
+    func configureMailComposeViewController() ->MFMailComposeViewController {
+        
+        let mail = MFMailComposeViewController()
+        mail.mailComposeDelegate = self as? MFMailComposeViewControllerDelegate
+        mail.setToRecipients(["support@amohi.org"])
+        mail.setSubject("iOS Amahi Anywhere")
+        mail.setMessageBody("<b>please wite the feedback</b>", isHTML: true)
+        return mail
+    }
+    func showSendMailErrorAlert()
+    {
+        let sendMailErrorAlert = UIAlertView(title: "could not send mail", message: "Your device couldnt send mail", delegate: self, cancelButtonTitle: "OK")
+        sendMailErrorAlert.show()
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        
+    }
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let section = indexPath.section
+        let row = indexPath.row
+        
+        if section == 0{
+            if row == 0 {
+                gotoConnections()
+            } }
+        else if section == 0 {
+            if row == 1{
+               goToVersion()
+            } }
+        else if section == 1 {
+            if row == 1 {
+                rateApp()
+            }
+        }
+        else if section == 1 {
+            if row == 0 {
+                let a = configureMailComposeViewController()
+                if MFMailComposeViewController.canSendMail()
+                {
+                    self.present(a, animated: true, completion: nil)
+                    
+                } else {
+                    self.showSendMailErrorAlert()
+                    
+                }
+                
+                
+            }
+            
+        }
+        else if section == 2 {
+            if row == 0 {
+                logout()
+            }
+        }
+        
+            print(indexPath.row)
+            tableView.deselectRow(at: indexPath, animated: true)
+        
+    }
+    
+    // MARK:- go to Connections
+    
+    func gotoConnections()
+    {
+        performSegue(withIdentifier: "goToConnections", sender: self)
+    }
+    
+    //Mark:- go to Version
```

Comment:
We can move all the Segue Identifiers to the Segue Identifiers Struct. That way it is easy to access all segues in the app. Check out [Utils/StringIdentifiers](https://github.com/amahi/ios/blob/master/AmahiAnywhere/AmahiAnywhere/Utils/StringIdentifiers.swift)

Reply:
Thanks for the review I will update the changes😊